src/content/self-hosted/install/releases.mdx

```diff
@@ -5,6 +5,34 @@ sidebar_label: Release Notes
 id: releases
 ---
 
+## 1.9.0
+12 June, 2023
+
+### Deprecation Notice
+
+- Support for Kubernetes [1.23](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.23.md) has been deprecated and will be removed in the next release.
+- Removed support for dedicated tls secrets for each ingress. 
```

Comment:
Does this require any other change in the documentation?

Reply:
i don't see the property tlsSecret was being documented
cc @pchico83

Reply:
We delete the docs a few weeks ago. I would make it clearer that this is for ingress created for the okteto installation

Reply:
@pchico83 as you have more context of it, could you propose a change for this item description?

Reply:
I would go with no documenting it in the release notes

Reply:
removed

Reply:
removed